src/pages/MyPage.jsx

```diff
@@ -1,20 +1,127 @@
-import { Avatar, Box, Button, Typography } from '@mui/material';
-import { priceToString } from 'utils/priceToString';
 import { PageContainer, PageContents } from 'components/Common';
 import MyPartyTab from 'components/MyParty/MyPartyTab';
-import { userInfo, parties } from 'constants/myPageDummyData';
-import { LogoutOutlined } from '@mui/icons-material';
+import useAsync from 'hooks/useAsync';
+import { getAllMyParty } from 'utils/api';
+import { useNavigate } from 'react-router';
+import { useEffect, useState } from 'react';
+import MyPageTitle from 'components/MyParty/MyPageTitle';
+import { useAuthState } from 'contexts/authContext';
+
+const LIMIT = 3;
+
+const initialState = {
+  parties: [],
+  lastPartyId: undefined,
+  loadedSize: LIMIT,
+  buttonDisabled: true,
+};
 
 const MyPage = () => {
-  const { userId, username, points } = userInfo;
-  console.log(userId);
+  const { username, points } = useAuthState();
+  const [onGoing, setOnGoing] = useState(initialState);
+  const [recruiting, setRecruiting] = useState(initialState);
+  const [finished, setFinished] = useState(initialState);
+
+  const [onGoingState] = useAsync(
+    getAllMyParty,
+    ['ONGOING', LIMIT, onGoing.lastPartyId],
+    [onGoing.lastPartyId],
+  );
+  const [recruitingState] = useAsync(
+    getAllMyParty,
+    ['RECRUITING', LIMIT, recruiting.lastPartyId],
+    [recruiting.lastPartyId],
+  );
+  const [finishedState] = useAsync(
+    getAllMyParty,
+    ['FINISHED', LIMIT, finished.lastPartyId],
+    [finished.lastPartyId],
+  );
+
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    if (onGoingState.value) {
+      setOnGoing({
+        ...onGoing,
+        parties: [...onGoing.parties, ...onGoingState.value.parties],
+      });
+    }
+  }, [onGoingState.value]);
```

Comment:
```suggestion
  useEffect(() => {
    if (onGoingState.value) {
      setOnGoing(prevOnGoing => ({
        ...prevOnGoing,
        parties: [...prevOnGoing.parties, ...onGoingState.value.parties],
      }));
    }
  }, [onGoingState.value]);
```
이부분 react-hooks/exhaustive-dep 관련 lint 에러 나타나는데 setOnGoing의 매개변수를 함수로 주면 사라질것 같습니다!